README.md

```diff
@@ -1,5 +1,25 @@
-# Getting Started with Create React App
-
+# Static Data format
+const User = {
+  firstName: "",
+  lastName: "",
```

Comment:
You can put type here, something like:

lastName: String,
socialURLs: Array(object)